pkgs/development/python-modules/debtcollector/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pbr
+, six
+, wrapt
+
+, doc8
+, fixtures
+}:
+
+buildPythonPackage rec {
+  pname = "debtcollector";
+  version = "2.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0ab8ic9bk644i8134z044kpcf6pvawqf4rq4xgv1p11msbs82ybq";
+  };
+
+  patches = [ ./tox-pass-system-pythonpath.patch ];
```

Comment:
tox overlaps with nix does, usually I just remove the tox.ini's before testing.

Reply:
Thanks for that, it makes it really easier !